### ISSUE TITLE
Make link post visually different

### ISSUE THREAD

**Issue by BeTmAsTeR** [probberechts/hexo-theme-cactus#294]
Right now it's impossible to discern a link post from a "standard" post

[example from the official demo page](https://i.imgur.com/ZNAnMlq.png)

It would be nice to do something to differentiate those type of posts from the others

**Comment by probberechts** [comment]
Yes, good suggestion. I am looking at other people who want this enhancement to make a pull request.
